modules/peers.js

```diff
@@ -225,7 +225,7 @@ __private.getByFilter = function(filter, cb) {
 		}
 	} else {
 		// Sort randomly by default
-		peers = shuffle(peers);
+		peers = __private.shuffle(peers);
```

Comment:
In `logic/peer` we used `_.shuffle`, so for consistency purpose we should use same shuffling method every where.